libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/configuration/hooks/useDependentDateFilterConfigurationState.ts

```diff
@@ -48,7 +47,7 @@ export function useDependentDateFilterConfigurationState(
         const commonDate = parseCommonDateFilter(commonDateFilter);
 
         return (filterElementsByDate ?? [])
-            .filter((parent) => !!parent.isCommonDate)
+            .filter((parent) => parent.isCommonDate)
```

Comment:
These were underlined by eslint in IDE as previously the type was `boolean | undefined` but was later changed to `boolean` and boolean re-casting operators remained in place even when they are no longer necessary.